### PR TITLE
[WIP]Adds the missing comment. solves issue #2147

### PR DESCRIPTION
**Pull request by pandyamarut**
fixes https://github.com/openebs/openebs/issues/2147
Signed-off-by: pandyamarut <pandyamarut@gmail.com>

### CODE REVIEW

File: types/profiles/v1/profiles.go [hunk 1]
```diff
@@ -14,8 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// This file deals with the labels, constants, defaults & utility
+// Package V1 deals with the labels, constants, defaults & utility
```
Comment (princerachit) [inline types/profiles/v1/profiles.go]: lower case `v1`

File: types/profiles/v1/profiles.go [hunk 2]
```diff
@@ -14,8 +14,9 @@ See the License for the specific language governing permissions and
...
 // functions related to the profiles
+
```
Comment (princerachit) [inline types/profiles/v1/profiles.go]: remove this empty line

File: types/profiles/v1/profiles.go [hunk 3]
```diff
@@ -45,6 +46,7 @@ import (
 //   size: 5G
 type ProvisionPolicyKey string
 
+//ReplicaCountPPK holds the provsion policy key for the provison/replica-count.
```
Comment (princerachit) [inline types/profiles/v1/profiles.go]: Add comment for every constant
```go
const (
        // ReplicaCountPPK holds the provsion policy key for the provison/replica-count.
	ReplicaCountPPK         ProvisionPolicyKey = "provision/replica-count"
        // ControllerCountPPK ...
	ControllerCountPPK      ProvisionPolicyKey = "provision/controller-count"
        //  ReplicaTolerationPPK ...
 	ReplicaTolerationPPK    ProvisionPolicyKey = "provision/replica-toleration"
        // ControllerTolerationPPK ...
 	ControllerTolerationPPK ProvisionPolicyKey = "provision/controller-toleration"
 )
```

File: types/v1/quantity.go [hunk 4]
```diff
@@ -13,7 +13,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// This is extracted from k8s.io/apimachinery/pkg/api/resource/quantity.go
+// Package v1 is extracted from k8s.io/apimachinery/pkg/api/resource/quantity.go
+
```
Comment (princerachit) [inline types/v1/quantity.go]: remove this empty line

**Comment by princerachit** [comment]
Please provide complete link to issue in the pr description

**Comment by pandyamarut** [comment]
@princerachit  please review.

**Comment by sonasingh46** [comment]
@pandyamarut 
Please address the comments provided by @princerachit

**Comment by sonasingh46** [comment]
@pandyamarut 
Closing due to inactivity. Please let us know if we need to work on the same.

**Pull request closed without merging**